### ISSUE TITLE
fasterq-dump uses incorrect exit codes for errors

### ISSUE THREAD

**Issue by Miserlou** [ncbi/sra-tools#146]
```
user@b400b2a23564:~/data_store/processor_job_1$ fasterq-dump /home/user/data_store/raw/TEST/SALMON/ERR1562482.sra -O /home/user/data_store/processor_job_1/
2018-09-11T19:19:47 fasterq-dump.2.9.1 err: invalid accession '/home/user/data_store/raw/TEST/SALMON/ERR1562482.sra'
user@b400b2a23564:~/data_store/processor_job_1$ echo $?
0
```

Should be -1 or similar.

**Comment by kwrodarmer** [comment]
Thanks for the report.

**Comment by klymenko** [comment]
The issue was fixed in 2.11.0 release. Please confirm.
